repositories/src/test/scala/cromwell/pipeline/datastorage/dao/utils/LiquibaseTestUtils.scala

```diff
@@ -0,0 +1,23 @@
+package cromwell.pipeline.datastorage.dao.utils
+
+import com.typesafe.config.{ Config, ConfigFactory }
+import cromwell.pipeline.database.LiquibaseUtils
+import cromwell.pipeline.utils.TestTimeout
+import slick.jdbc.JdbcProfile
+
+import scala.concurrent.Await
+
+object LiquibaseTestUtils extends TestTimeout {
+
+  val config: Config = ConfigFactory.load()
+  private val changeLogCleanTablesResourcePath: String =
+    config.getString("database.liquibase.changeLogCleanTablesResourcePath")
```

Comment:
IMO, it's not a good thing that `LiquibaseTestUtils` knows how to read `config` and how raw path to `changeLogCleanTablesResourcePath` looks like. We should either merge content of this class with `PostgreTablesCleaner`, or at least take this params from outside.
@ilyaonishenko what do you think?

Reply:
Fixed